### PR TITLE
Draft: General edits

### PR DESCRIPTION
**Pull request by CitMC**
Documentation content update across modules

### Change scope

The changes in this Pull Request include (please tick all that apply):

* [x] Small language/grammar fixes
* [x] Small content fixes 
* [ ] Addition of new content
* [ ] Sample code/command updates
* [ ] Platform fixes
* [ ] Other

### CODE REVIEW

**Review by coldice — APPROVED** [review]

File: academy/1-what-is-cosmos/cosmos-ecosystem.md [hunk 1]
```diff
@@ -72,9 +72,9 @@ The Cosmos network focuses on an ecosystem for easy blockchain development that
 
 Before Cosmos came along, developing a whole new chain was much more difficult and expensive than building a smart contract. Now, with the Cosmos SDK, entirely flexible, secure, high performance, and sovereign **application-specific blockchains** can be developed. To allow this, building modular, adaptable, and interchangeable open-source development tools is at the center of Cosmos' mission.
 
-The main aim of the Cosmos network is to provide an **ecosystem for easy blockchain development** based on the Tendermint BFT and the Inter-Blockchain Communication protocol (IBC) with the so-called [Cosmos SDK](https://v1.cosmos.network/sdk).
+The main aim of the Cosmos network is to provide an **ecosystem for easy blockchain development** based on the Tendermint BFT and the Inter-Blockchain Communication Protocol (IBC) with the so-called [Cosmos SDK](https://v1.cosmos.network/sdk).
```
Comment (xavierlepretre) [inline academy/1-what-is-cosmos/cosmos-ecosystem.md:75]: The previous "protocol" were not capitalised.

File: academy/3-my-own-chain/game-winner.md [hunk 2]
```diff
@@ -277,14 +277,6 @@ $ checkersd tx checkers play-move 1 1 2 2 3 --from $bob
...
 ```
 
-This should show:
-
-```
-...
-  winner: "*"
-...
-```
-
```
Comment (xavierlepretre) [inline academy/3-my-own-chain/game-winner.md:287]: This is not duplicate content. This is to show that this part of the content has not changed after a move.

File: academy/1-what-is-cosmos/cosmos-ecosystem.md (hunk 1 above)
Comment (CitMC) [reply academy/1-what-is-cosmos/cosmos-ecosystem.md:75]: It is a styling consistency change we are implementing.

File: academy/3-my-own-chain/game-winner.md (hunk 2 above)
Comment (CitMC) [reply academy/3-my-own-chain/game-winner.md:287]: Please include it again. I was really unsure about that one.

File: academy/1-what-is-cosmos/cosmos-ecosystem.md (hunk 1 above)
Comment (xavierlepretre) [reply academy/1-what-is-cosmos/cosmos-ecosystem.md:75]: I mean, the previous word "protocol" in this document was not capitalised.

Comment (CitMC) [reply academy/1-what-is-cosmos/cosmos-ecosystem.md:75]: Didn't see that one. I'll give the complete content another sweep for the final QA, and have a look specifically at that.

File: academy/3-my-own-chain/game-winner.md (hunk 2 above)
Comment (CitMC) [reply academy/3-my-own-chain/game-winner.md:287]: Included in https://github.com/cosmos/sdk-tutorials/pull/1169/files#diff-cf9cbdadfc7b04d5c082ee3dd04afc4c491f70826d5f8526750293020d33c748R384-R390 again.

**Pull request merged**